src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -167,32 +166,32 @@ def waitForQualityGateResult(timeoutInMinutes) {
   }
 }
 
-def buildAndPushContainerImage(credentialsId, registry, imageName, tag) {
+def buildAndPushContainerImage(credentialsId, registry, projectName, tag) {
```

Comment:
For the test image, I think `projectName` makes sense as it's used to set the project property.  In this case we are setting the name of the image so `imageName` seems clearer as to our intent.

Reply:
Agreed, amended.